src/media/js/site/login.js

```diff
@@ -25,10 +25,9 @@ export function loginRequired(Component, LoginHandler, group,
   class AuthenticatedComponent extends React.Component {
     render() {
       const isLoggedIn = !!this.props.user.token;
-      const hasPermission = (process.env.NODE_ENV === 'test' ||
-                             _checkPermissions(this.props.user, group));
+      const hasPermission = _checkPermissions(this.props.user, group);
 
-      if (isLoggedIn && hasPermission) {
+      if (process.env.NODE_ENV === 'test' || isLoggedIn && hasPermission) {
```

Comment:
I think it's very clean and manageable, much better than global runtime flags like we have for Fireplace. When the bundle is built, the clause will get completely removed (```'test' === 'test'``` => ```true``` => nothing).